gp-templates/translations.php

```diff
@@ -75,18 +75,55 @@
 	<?php
 	$filter_links = array();
 	$filter_links[] = gp_link_get( $url, __( 'All', 'glotpress' ) );
-	$untranslated = gp_link_get( add_query_arg( array('filters[status]' => 'untranslated', 'sort[by]' => 'priority', 'sort[how]' => 'desc'), $url ), __( 'Untranslated', 'glotpress' ) );
-	$untranslated .= '&nbsp;('.gp_link_get( add_query_arg( array('filters[status]' => 'untranslated', 'sort[by]' => 'random'), $url ), __( 'random', 'glotpress' ) ).')';
-	$filter_links[] = $untranslated;
+
+	$untranslated_filters = array(
+		'filters[status]' => 'untranslated',
+		'sort[by]'        => 'priority',
+		'sort[how]'       => 'desc',
+	);
+
+	$filter_links[] = gp_link_get(
+		add_query_arg( $untranslated_filters, $url ),
+		// Translators: %d is the untranslated strings count for the current translation set.
+		sprintf( __( 'Untranslated&nbsp;[%d]&nbsp;(random)', 'glotpress' ), $translation_set->untranslated_count() )
```

Comment:
We should use `number_format_i18n()` for numbers (and `%s` for the placeholder).